.github/codeql/codeql-config.yml

```diff
@@ -0,0 +1,3 @@
+query-filters:
+  - exclude:
+      id: go/disabled-certificate-check
```

Comment:
Just for information, this is to ignore every warning about `InsecureSkipVerify: true`

https://codeql.github.com/codeql-query-help/go/go-disabled-certificate-check/#recommendation

Reply:
```suggestion
      # this is to ignore every warning about InsecureSkipVerify: true
      id: go/disabled-certificate-check
```